### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by Blueberryy**

### CODE REVIEW

File: Russian.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    "command.notinlobby": "Эту команду можно использовать только в лобби.",
+    "command.tell.receiver": "{%sender} прошептал вам: {%message}",
+    "command.tell.sender": "Вы прошептали {%receiver}: {%message}",
+    "command.tell.yourself": "Вы не можете послать сообщение самому себе.",
+    "command.tp.specify": "Вы должны указать игрока.",
+    "command.tphere.accepted.receiver": "Принят запрос {%sender} на телепортацию.",
+    "command.tphere.accepted.sender": "{%receiver} принял ваш запрос на телепортацию к вам.",
+    "command.tphere.declined.receiver": "Отклонён запрос {%sender} на телепортацию.",
+    "command.tphere.declined.sender": "{%receiver} отклонил вашу запрос на телепортацию к вам.",
+    "command.tphere.noperm": "§cУ вас нет разрешения отправлять запрос другим игрокам телепортироваться к вам. Купите §l§aИЗУМРУДНЫЙ§r §cили §l§bЛЕГЕНДАРНЫЙ§r §c ранг на §bngmc.co/store§c, чтобы отправить!",
```
Comment (hampuskraft) [inline Russian.json:19]: Rank names should be untranslated to avoid confusion.

File: Russian.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    "command.tphere.norequest": "Этот игрок не отправил запрос на телепортацию к нему.",
+    "command.tphere.receive": "{%sender} отправляет запрос на телепортацию к нему! Используйте §6/tphere accept {%0}, чтобы принять запрос.",
+    "command.tphere.send": "Ваш запрос на телепортацию {%receiver} к вам отправлен.",
+    "command.tpto.accepted.receiver": "{%sender} принимает запрос телепортации к вам.",
+    "command.tpto.accepted.sender": "{%receiver} принимает ваш запрос на телепортацию к нему.",
+    "command.tpto.declined.receiver": "{%sender} отклоняет запрос телепортации к вам.",
+    "command.tpto.declined.sender": "{%receiver} отклоняет ваш запрос на телепортацию к нему.",
+    "command.tpto.noperm": "§cУ вас нет разрешения на запрос телепортации к другим игрокам. Купите §l§aИЗУМРУДНЫЙ§r §cили §l§bЛЕГЕНДАРНЫЙ§r §c ранг на §bngmc.co/store§c, чтобы отправить!",
```
Comment (hampuskraft) [inline Russian.json:27]: Rank names should be untranslated to avoid confusion.

File: Russian.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    "party.ended.host": "Вы успешно завершили игру с группой.",
+    "party.invited": "Вы пригласили {%invited} в вашу группу.",
+    "party.join.game": "В настоящее время вы находитесь в группе. Подождите, пока организатор группы решит, в какую игру играть!",
+    "party.kicked": "Вас выгнали из группы {%kicker}.",
+    "party.kicked.message": "{%kicker} выгоняет {%kicked} из группы.",
+    "party.leave.player": "Вы покинули группу.",
+    "party.max": "Вы не можете пригласить в свою группу более {%max} игроков!",
+    "party.still.in.game": "{%member} ещё участвуют в матче. Дождитесь их, прежде чем присоединиться к другому!",
+    "party.welcome": "Добро пожаловать в группу {%host}!",
+    "pets.ride.noperm": "§cУ вас нет разрешения ездить на питомцах и управлять ими.\\nКупите §l§cТИТАНОВЫЙ§r §cранг на §bngmc.co/store§c, чтобы ездить на них!",
```
Comment (hampuskraft) [inline Russian.json:87]: Rank names should be untranslated to avoid confusion.

**Review by hampuskraft — CHANGES_REQUESTED** [review]
Thanks for your contribution! If you could just resolve the comments (rank names should be untranslated) the PR is ready to merge :shipit:

**Comment by Blueberryy** [comment]
Should be solved now

**Review by hampuskraft — APPROVED** [review]

**Comment by Driesboy** [comment]
Thanks for your contribution

**Pull request merged**